deployment/kubernetes/flood-api.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flood-api-deployment
+  namespace: apps-flood-api
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: flood-api
+  template:
+    metadata:
+      namespace: apps-flood-api
+      labels:
+        app: flood-api
+    spec:
+      containers:
+      - image: ghcr.io/openearthplatforminitiative/flood-api:0.3.0
+        name: flood-api
+        ports:
+        - containerPort: 8080
+        env:
+        - name: DB_NAME
+          value: flood_data
+        - name: DB_ENDPOINT
+          valueFrom:
+            secretKeyRef:
```

Comment:
Pods will fail to start until the secret is created, so I did not need an init-container yet